src/Interfaces/AbstractZonotope.jl

```diff
@@ -419,6 +419,18 @@ https://math.stackexchange.com/q/3356460
 To avoid cumulative sum from both directions separately, we build a 2d index matrix
 to sum generators for both directions in one matrix-vector product.
 """
+
+@inline function _angles(point::AbstractVector{N})  where{N<:Real}
+    (atan(point[2], point[1]) /π * 180 + 360)%360
+end
+
+function _enum(G::AbstractMatrix{N}, sorted::Bool=true) where{N<:Real}
```

Comment:
Can you use a more specific function name, please? :)
It helps to understand the functionality a bit and to avoid accidentally calling it in other contexts.
We often just use the caller name plus some addition, like `_vertices_list_zonotope_enum`, but other names are also fine.

Reply:
see https://github.com/JuliaReach/LazySets.jl/pull/2303, which prepares the stage for the new 2d method here https://github.com/JuliaReach/LazySets.jl/pull/2303/files#diff-42ebbc16f90bae7a08e04f097c7aeb9eR506